### DISCUSSION TITLE
Define a schema from JSON

### DISCUSSION THREAD

**Discussion by wertania** [open-circle/valibot#885] [Q&A]
Hi,

Thanks to this great library.
Is it possible to create a validation schema from a JSON object like this?

```typescript
const MySchema = v.fromJSON({
  "kind": "schema",
  "type": "string",
  "expects": "string",
  "async": false
})
```

**Comment by fabian-hiller** [comment]
No, not yet. In the long run, I plan to add an option to serialize a schema (this probably already works with `JSON.stringify` for most schemas) and deserialize it. But at the moment, other things have a much higher priority for the library.

What is your use case? Feel free to tell us more and provide some details.
